ember-scoped-css/src/scoped-babel-plugin.js

```diff
@@ -6,22 +6,78 @@ import generateHash from './lib/generateRelativePathHash.js';
 import getClassesTagsFromCss from './lib/getClassesTagsFromCss.js';
 import rewriteHbs from './lib/rewriteHbs.js';
 
+function isRelevantFile(state) {
+  /**
+   * Mostly pods support.
+   * folks need to opt in to pods, because every pods app can be configured differently
+   */
+  let roots = ['/components/', ...(state.opts?.additionalRoots || [])];
+  let filename = state.file.opts.filename;
+
+  if (!roots.some((root) => filename.includes(root))) {
+    return;
+  }
+
+  return true;
+}
+
 export default () => {
+  /**
+   * - This can receive the intermediate output of the old REGEX-based <template> transform:
+   *   ```
+   *   import { scopedClass } from 'ember-scoped-css';
+   *
+   *   __GLIMMER_TEMPLATE(`
+   *     original <template> innards here
+   *   `);
+   *   ```
+   *   - the import is optional, though, required for type-checking in gts (so we don't mess with globals)
+   *   - the babel-plugin-ember-template-compilation step has not run yet,
+   *     else we'd see precompileTemplate and setComponentTemplate (and more imports)
+   *
+   *   - note that in ember-template-imports' implementation, the file changes
+   *     after `ImportDeclaration` visitors have ran, and by the time we see
+   *     CallExpressions, we have the familiar `setComponentTemplate`
+   */
   return {
     visitor: {
+      ImportDeclaration(path, state) {
```

Comment:
part 2 of the fix is here, we need to remove the import -- this is mostly duplicate code, but any overlap between the AST structures of Babel and Recast seems to be cooincidence, and I couldn't find anything saying they must be the same. 
The two AST visitor tools also have different APIs for removing / entering nodes -- an abstraction could be made -- not sure if worth it yet.